bsip-0076.md

```diff
@@ -0,0 +1,88 @@
+the value of the token is distorted, thus, forcing BTS token holders who have their BTS token trapped in
+debt positions as collateral to sell their collateral at the distorted price via margin calls or
+force-settlements is unfair.
+The same situation applies to non-committee-owned smart coins as well.
+
+Smartcoins are products of the BitShares platform, their rules are defined by the platform
+for the benefit of the platform and the BTS token holders.
+If a product brings more harm than benefit to the platform, its rules need to be optimized,
+under extreme conditions, the product needs to be halted.
+For smartcoins, when the cost to maintain the peg is too high to afford, it's time to give up the peg.
+
+# Rationale
+
+Ideally a smartcoin owner should be able to define a set of conditions about when to give up the peg
+and how to de-peg, but it needs time to design and implement. Before the final solution got worked out,
+it's up to the price feed producers to execute the will for the asset owner on whether to maintain
+the peg at a given point and how to de-peg. For committee-owned smartcoins, the owner is actually
+all BTS holders who could express their will by voting, the price feed producers are the active witnesses.
+
+# Specification
+
+The BTS token holders define a threshold, if the market trading price of BTS is below the threshold,
+witnesses should feed the threshold but not the market trading price, thus would effectively break
+the peg as intended. When the market trading price of BTS is above the threshold, witnesses should
+feed the market trading price, so the peg will be restored automatically. The mechanism is similar to
+the Global-Settlement Protection mechanism as described in [BSIP 58](bsip-0058.md).
+
+Changes to the threshold will be decided on individual committee-owned smartcoins by voting, each time
+one change is issued 2 poll worker proposals will be created standing FOR and AGAINST the change.
+
+For example, to decide whether to change the price feed threshold of bitCNY to 0.2 CNY each BTS,
+2 poll worker proposals will be created for voting:
+
+Poll-BSIP**-Change bitCNY feed threshold [from the previous value when applicable] to 0.2 CNY per BTS
+
+Poll-BSIP**-Not change bitCNY feed threshold [from the previous value when applicable] to 0.2 CNY per BTS
+
+If the voting confirm the change, committee will announce the change at least 3 days before the change
+is implemented by witnesses.
+
+# Discussion
+
+Some people think it's the debt position holders' responsibility to always maintain their
+collateral ratio to a fair level. However, when the collateral's price can drop to zero due to
+unfair manipulation, it's not possible to always maintain a good CR.
+Higher or lower, there is a point that the peg would break.
+
+This BSIP may negatively impact smartcoin holders since their holdings will likely devalue when
```

Comment:
There is no 'may', this BSIP did directly negatively impact smartcoin holders holding value, they now hold fraudulently valued assets instead of pegged assets which always adhere to their originally designed contract. CNY & USD experienced up to 30% devaluation in the last month after the forced implementation of this BSIP (prior to any related WP activating).